ghcide/src/Development/IDE/Types/HscEnvEq.hs

```diff
@@ -76,22 +76,25 @@ newHscEnvEqWithImportPaths envImportPaths hscEnv deps = do
         -- compute the package imports
         let pkgst   = unitState hscEnv
             depends = explicitUnits pkgst
-            targets =
-                [ (pkg, mn)
+            modules =
+                [ m
                 | d        <- depends
                 , Just pkg <- [lookupPackageConfig d hscEnv]
-                , (mn, _)  <- unitExposedModules pkg
+                , (modName, maybeOtherPkgMod) <- unitExposedModules pkg
+                , m <- pure $ case maybeOtherPkgMod of
```

Comment:
Why not use `let` here? Something like
```suggestion
                , let m =  case maybeOtherPkgMod of
```

Reply:
I'm not used to using list comprehensions so much, so it didn't occur to me